tests/integration/components/copy-button-test.js

```diff
@@ -0,0 +1,20 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | copy-button', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
```

Comment:
would like a test case for the passed-in click-handler.

Reply:
Me too! Unfortunately if you try and write to the clipboard in a text context it dies because you don't have permission to do this. It's.... annoying.

Reply:
okely dokely.